cpp/oneapi/dal/backend/interop/table_conversion.hpp

```diff
@@ -16,11 +16,16 @@
 
 #pragma once
 
-#include <daal/include/data_management/data/homogen_numeric_table.h>
-
 #include "oneapi/dal/data/accessor.hpp"
 #include "oneapi/dal/data/table_builder.hpp"
 
+#ifdef ONEAPI_DAL_DATA_PARALLEL
+    #define DAAL_SYCL_INTERFACE
+    #define DAAL_SYCL_INTERFACE_USM
+    #include <daal/include/data_management/data/numeric_table_sycl_homogen.h>
+#endif
+#include <daal/include/data_management/data/homogen_numeric_table.h>
+
```

Comment:
Move this section before oneapi/dal includes

Reply:
Applied